src/kapil/Switch/BankSwitch.java

```diff
@@ -0,0 +1,165 @@
+/*Assignment - 15 : 9th April'2021
+
+A person has total 4 accounts in ICICI, HDFC, BOB, SBI one in each.
+a) Person can debit or credit amount from respective bank.
+Hint : Switch case is expected here. [Please watch today's recording again, in case of confusion.]
+
+b) A Method to show total balance [ICICI balance + HDFC balance + BOB balance + SBI Balance]
+
+c) A Method to show total credit, debit operations
+output : Total credit operation - 7 times
+Total debit operation - 5 times
+
+d) A Method to show individual bank credit operations.
+output : ICICI bank Credit Operation - 2
+HDFC bank Credit Operation - 2
+SBI bank Credit Operation - 1
+BOB bank Credit Operation - 4
+
+e) A Method to show individual bank debit operations.
+output : ICICI bank Debit Operation - 2
+HDFC bank Debit Operation - 2
+SBI
+
+*/
+package kapil.Switch;
+
+public class BankSwitch {
+       int iciciBalance = 40000;
+       int bobBalance = 30000;
+       int sbiBalance = 20000;
+       int hdfcBalance = 10000;
+       
+   int iciciDebitCount = 0, iciciCreditCount = 0,bobDebitCount = 0, bobCreditCount = 0, sbiDebitCount = 0, sbiCreditCount = 0, hdfcDebitCount=0, hdfcCreditCount=0;    
+   int totalDebitCount, totalCreditCount;
+   
+   
+   void operation1(String bank, String op, int amt) {
+	   switch(op) {
+	     case"Debit" :
+		   switch (bank) {
+		   case "icici" :
+	    	   if(iciciBalance>= amt) {
+	    		   iciciBalance = iciciBalance - amt;
+	          	   iciciDebitCount = iciciDebitCount + 1;	
+				   totalDebitCount = totalDebitCount + 1;
+				   System.out.println("icici debit amount is "+iciciBalance);
+				   }
+		
+	    	   else
+	    		   System.out.println("Balance is not sufficient ");
+	    	   break;
```

Comment:
Please make sure your indentation is right, I will allow this time, from the next time I will not going to review code if code does not have indentation.
Control + Shift + F is for code indentation in eclipse